orchagent/crmorch.h

```diff
@@ -0,0 +1,92 @@
+#pragma once
+
+#include <thread>
+#include <chrono>
+#include <map>
+#include "orch.h"
+#include "port.h"
+
+extern "C" {
+#include "sai.h"
+}
+
+using namespace std;
+
+enum class CrmResourceType
+{
+    CRM_IPV4_ROUTE,
```

Comment:
Previously I tried to remove CRM prefix, but got compilation error because, e.g. IPV6_NEXTHOP is already defined in some Linux header file. Due to this error was observed - "expected identifier before numeric constant".
So, I believe it is better to leave it is as is and I also added "CRM" prefixes for CrmThresholdType.